app/javascript/articles/components/SaveButton.jsx

```diff
@@ -3,9 +3,28 @@ import PropTypes from 'prop-types';
 import { articlePropTypes } from '../../src/components/common-prop-types';
 
 export class SaveButton extends Component {
+  constructor() {
+    super();
+    this.state =
+      {
+        collectionDropboxValue: 'Default Collection',
+        allCollectionTitles: []
+      }
+  }
+
   componentDidMount() {
+    fetch('http://localhost:3000/api/user_collections',  {credentials: 'include'})
```

Comment:
This will need to be broken into a new separate function which will live in an apiCalls.js file for testing purposes. Otherwise, we will be unable to test our fetch since it is defined in the `componentDidMount` method.